doc/input-output-reference/src/overview/group-variable-refrigerant-flow-equipment.tex

```diff
@@ -74,13 +74,13 @@ \subsubsection{Inputs}\label{inputs-051}
 
 This numeric field defines the cooling coefficient of performance at rated conditions. The cooling coefficient of performance includes compressor power and condenser fan power. This COP value does not account for impacts due to the supply air fan. The nominal heat pump cooling COP must be greater than 0. If this field is left blank, a default coefficient of performance of 3.3 is assumed.
 
```

Comment:
A user was confused on the use and interpretation of the next few input fields. A water-cooled VRF will not use the outdoor air temp to determine operation. Generalize the field name and describe in text.